### DISCUSSION TITLE
Machine Learning and AI in Computational Mechanics

### DISCUSSION THREAD

**Discussion by mokhairymahmoud** [KratosMultiphysics/Kratos#9124] [Q&A]
Lately, it came to my attention the growth of machine learning and AI in all sciences including computational mechanics. I was wondering how can these techniques be used in computational mechanics. So, to make it short, I would like to know your point of view on some important point:

1) Do you think that neural networks could in the future replace the standard methods like FEM ?

2) Where do you think Machine learning and AI fit in the domain of computational mechanics ?

**Comment by loumalouomega** [comment]
I recommend taking a look to https://developer.nvidia.com/simnet

And I know there is someone working in this subject in Kratos

**Comment by loumalouomega** [comment]
>     1. Do you think that neural networks could in the future replace the standard methods like FEM ?

>     2. Where do you think Machine learning and AI fit in the domain of computational mechanics ?

I think that the purpose of these techniques is working in problems were deterministic definitions can not be applied. FEM is for physics, so in this sense you don't need them. You may be interested in using it to enrich the solution as a pre/postprocessing operation as style transfer:

https://www.youtube.com/watch?v=leoRHsBsv6Q

Or you be also interested to perform a coarse simulation and enrich using these techniques. It could be also interesting for computation in the edge for precomputed models, among others, but of course will never replace a full simulation as these techniques may introduce errors that we are not able to detect in complex cases (because it is a black box, and we don't know what it does). Conclusion, never a replacement, only a complement, or may be a replacement in some very specific and very known cases.

**Comment by loumalouomega** [comment]
Some more info: https://hal.archives-ouvertes.fr/hal-03327818/document
